### PR TITLE
Move to Electron & Squirrel, Remove ffmpeg

### PR DESCRIPTION
**Pull request by felixrieseberg**
This is a slightly bigger PR, so let's list the changes individually:

#### Electron
Glorious Electron! I first built `ember-electron` (which is really just a fancy copy of `ember-cli-nwjs`), meaning that we can now run our stuff in Electron. The following commands work:

```
ember electron
ember electron:test
ember electron:test --server
```

Most things should be identical, but if we find quirks, they should be easy to fix.

#### Squirrel
Squirrel is a fancy installer that includes delta-updates, meaning that the application can update itself. Bonus: Installation is really pretty. I haven't done too much on this front yet, but the installer generation already works.

#### Built Tasks
Nothing exciting here, I simply changed our Grunt tasks to match Electron best practices.

### CODE REVIEW

**Comment by felixrieseberg** [comment]
@sedouard: I think that I got most bases covered, but I wouldn't be shocked if we find a few quirks down the road. I'd be thankful if you could lend me a second pair of :eyes: to make sure that I didn't forget something super obvious.

File: package.json [hunk 1]
```diff
@@ -68,33 +72,17 @@
...
-    "icon": "./public/icon/ase.png"
-  },
+  "main": "electron.js",
   "dependencies": {
     "azure-storage": "^0.4.3",
     "fs-extra": "^0.22.1",
-    "grunt-nw-builder": "^2.0.0",
-    "nw": "0.12.3",
-    "open": "0.0.5"
+    "memorystream": "^0.3.1"
```
Comment (sedouard) [inline package.json]: I can't find where you are using the `memorystream` module in the app. Is this a really a runtime dependency?

File: postcompile/osx/licenseDMG.py [hunk 2]
```diff
@@ -1,163 +0,0 @@
-#! /usr/bin/env python
-"""
```
Comment (sedouard) [inline postcompile/osx/licenseDMG.py]: Yay!

File: package.json (hunk 1 above)
Comment (felixrieseberg) [inline package.json]: I couldn't find it either, but you had a copy-task for it - if you're not using it anymore, we should get rid of it!

File: vendor/node-webkit/reload.js [hunk 3]
```diff
@@ -1,14 +0,0 @@
-/* jshint browser: true */
```
Comment (sedouard) [inline vendor/node-webkit/reload.js]: Does ctrl+save trigger a reload still?

Comment (felixrieseberg) [inline vendor/node-webkit/reload.js]: Yep, managed in the addon.

**Comment by sedouard** [comment]
@felixrieseberg I think your CI changes a few days ago prevents the actual application from being packaged. (not zipped, but packaged). It would be good to keep this in CI, for devops changes like these.

**Comment by felixrieseberg** [comment]
I just merged another PR that splits the task up into two - we're now testing the packaging, but only attempting to deploy when we're on the right branch.

**Pull request merged**